Documentation/install/upgrade.rst

```diff
@@ -149,7 +149,7 @@ Step 2: Option A: Regenerate deployment files with upgrade compatibility (Recomm
 
 `Helm` can be used to generate the YAML files for deployment. This allows to
 regenerate all files from scratch for the new release. By specifying the option
-``--set config.upgradeCompatibility=1.7``, the generated files are guaranteed
+``--setupgradeCompatibility=1.7``, the generated files are guaranteed
```

Comment:
`setupgradeCompatibility` here and at least the next couple of instances needs a space between `set` and `upgradeCompatibility`. Probably worth doing a quick grep for `set[^ ]` to check if there are other patterns like this in the PR.

Reply:
done and thanks